### PR TITLE
Added a new field

### LINKED ISSUE #497: Add addl date on Team
We need a good way to report when Team Training actually starts which can be pretty different than their agreement date or date the program was assigned. 

Let's add "Training Start Date". it will be manually set. Let's put it in the page layout where this red box is.
image

### PR DESCRIPTION
**Pull request by mguillendev**
# Critical Changes

# Changes

- Created the "Training Start Date" custom field on Team object and located in the correct position in the team layout. 
- Added the permission set to view and edit the field for Altas Super Users and Atlas Teams.

# Issues Closed

#497

### CODE REVIEW

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml [hunk 1]
```diff
@@ -88,11 +88,6 @@
         <field>CategoryRule__c.Group__c</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
```
Comment (deb761) [inline force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: Ooops!  We need to leave this permission here.

**Review by deb761 — CHANGES_REQUESTED** [review]
Looks good except for the accidentally-deleted permission.

File: force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml (hunk 1 above)
Comment (deb761) [reply force-app/main/default/permissionsets/AtlasSuperUser.permissionset-meta.xml:91]: My bad.  You're removing a duplicate.

**Review by deb761 — APPROVED** [review]

**Comment by atlasjen** [comment]
looks good.

**Pull request merged**